<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>528</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LEO Random Ramp</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>17</y>
     <width>281</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_node_name">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Selected Node: None</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_update_node">
      <property name="text">
       <string>Update Selected Node</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="dropdown_parm"/>
    </item>
    <item>
     <widget class="QComboBox" name="dropdown_basis">
      <item>
       <property name="text">
        <string>Linear</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Constant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CatmullRom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MonotoneCubic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bezier</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BSpline</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hermite</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Number Of Points:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="value_samples">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radiobutton_override">
         <property name="text">
          <string>Override Current Values</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radiobutton_adjust">
         <property name="text">
          <string>Adjust Current Values</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="override_box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>Override Current values</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QDoubleSpinBox" name="value_set_max">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="value_set_min">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="adjust_box">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="title">
       <string>Adjust Current values</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>134</width>
         <height>93</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QDoubleSpinBox" name="value_adjust_min">
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="value">
             <double>-0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="value_adjust_max">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkbox_adjust_clamp">
          <property name="text">
           <string>Clamp Value To 0 - 1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radiobutton_add">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radiobutton_mult">
          <property name="text">
           <string>Multiply</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_apply">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dropdown_parm</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radiobutton_override</sender>
   <signal>toggled(bool)</signal>
   <receiver>override_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiobutton_adjust</sender>
   <signal>toggled(bool)</signal>
   <receiver>adjust_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
